### PR TITLE
fix(console): Use correct set handlers when updating passwordless message-texts

### PR DESCRIPTION
**Pull request by JohnnyCrazy**
When updating passwordless message-texts, they were not persisted. When reloading the page, the changes were gone. Turns out they use `get...` method, although they should use the `set...` version.

Discord Thread: https://discord.com/channels/927474939156643850/1072546283639799839

```[tasklist]
### Definition of Ready
- [x] I am happy with the code
- [x] Short description of the feature/issue is added in the pr description
- [ ] PR is linked to the corresponding user story
- [ ] Acceptance criteria are met
- [ ] All open todos and follow ups are defined in a new ticket and justified
- [ ] Deviations from the acceptance criteria and design are agreed with the PO and documented.
- [x] No debug or dead code
- [ ] Critical parts are tested automatically
- [ ] Where possible E2E tests are implemented
- [ ] Documentation/examples are up-to-date
- [ ] All non-functional requirements are met
- [ ] Functionality of the acceptance criteria is checked manually on the dev system.
```

### CODE REVIEW

File: console/src/app/modules/policies/message-texts/message-texts.component.ts [hunk 1]
```diff
@@ -596,7 +596,7 @@ export class MessageTextsComponent implements OnInit, OnDestroy {
           return handler((this.service as ManagementService).setCustomDomainClaimedMessageCustomText(this.updateRequest));
         case MESSAGETYPES.PASSWORDLESS:
           return handler(
-            (this.service as ManagementService).getCustomPasswordlessRegistrationMessageText(this.updateRequest),
+            (this.service as ManagementService).setCustomPasswordlessRegistrationMessageCustomText(this.updateRequest),
           );
         case MESSAGETYPES.PASSWORDCHANGE:
           return handler((this.service as ManagementService).getCustomPasswordChangeMessageText(this.updateRequest));
```
Comment (JohnnyCrazy) [inline console/src/app/modules/policies/message-texts/message-texts.component.ts:602]: This is also wrong, but it seems like it involves creating a new GRPC method and all. Not sure if I can invest the time figuring this out rather soon.

**Comment by hifabienne** [comment]
@peintnermax can you have a look at this?

**Review by peintnermax — APPROVED** [review]

File: console/src/app/modules/policies/message-texts/message-texts.component.ts (hunk 1 above)
Comment (peintnermax) [reply console/src/app/modules/policies/message-texts/message-texts.component.ts:602]: Thx for the fix :) Do you mean the typing of the updateRequest?

Comment (JohnnyCrazy) [reply console/src/app/modules/policies/message-texts/message-texts.component.ts:602]: Hi, sorry for the long silence. Yes, the GRPC typing for updating the password change texts were missing and I didn't have time to invest further at that point.

**Pull request merged**